metabase/metabase
# Fix visualization settings for dark brand colors

Fixes https://github.com/metabase/metabase/issues/23759

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Appearance
- Change the brand color to a dark one, e.g. `#663725`
- Go to a saved question and click on `Visualization`
- Check that each visualizuation icon can be seen
- Click on `Line`
- Check that settings tabs have visible text

<img width="621" alt="Screenshot 2022-07-08 at 16 29 48" src="https://user-images.githubusercontent.com/8542534/178001889-2d43af93-9c11-405a-ab69-d7e4b08eba3f.png">
<img width="727" alt="Screenshot 2022-07-08 at 16 29 59" src="https://user-images.githubusercontent.com/8542534/178001912-e8387d4a-0c6b-4dbf-af27-f48c3f3fbe28.png">
